src/openrct2/rct2/RCT2.cpp

```diff
@@ -142,8 +142,8 @@ namespace RCT2
     bool RCT2TrackTypeIsBooster(uint8_t rideType, uint16_t trackType)
     {
         // Boosters share their ID with the Spinning Control track.
-        return rideType != RIDE_TYPE_SPINNING_WILD_MOUSE && rideType != RIDE_TYPE_STEEL_WILD_MOUSE
-            && trackType == TrackElemType::Booster;
+        const auto& rtd = GetRideTypeDescriptor(rideType);
+        return !rtd.HasFlag(RIDE_TYPE_FLAG_NO_BOOSTERS) && trackType == TrackElemType::Booster;
```

Comment:
This is in the RCT2 import layer, which will stick with ride type checks. At some point, we will make the ride type enum specific to RCT2.h.